DataCollection/src/Services/Controllers/WaterbirdForagingSurveyController.cs

```diff
@@ -40,7 +40,7 @@ public IHttpActionResult Post([FromBody]WaterbirdForagingModel input)
         {
             if (input == null)
             {
-                throw new ArgumentNullException("input");
+                return BadRequest("null input");
```

Comment:
I tried sending no JSON data to the controller. I got back a status code with an HTML-based "argument null" error, rather than a proper API error response. I had never thought to test that before. Therefore we need to start changing all controller methods to `return BadRequest("null input")` instead of throwing an `ArgumentNullException`.